data-source/jsonnet/common/blocks/individual/personal-details/name.jsonnet

```diff
@@ -25,8 +25,13 @@ local question(title) = {
     {
       id: 'last-name',
       label: 'Last name',
-      mandatory: false,
+      mandatory: true,
       type: 'TextField',
+      validation: {
+        messages: {
+          MANDATORY_TEXTFIELD: 'Please enter a last name to continue',
```

Comment:
The 'Please' needs to dropped from these error messages (checked with @charlotte-hirst and Donna).

Reply:
Removed from messages in that file. I assume all the labels elsewhere are still okay to use `Please`?

Reply:
Good spot, probably not. I'd say it shouldn't block this going in, but should be raised with @charlotte-hirst.